pkg/plugins/trivy/plugin.go

```diff
@@ -232,3 +236,17 @@ func (p *plugin) parseImageRef(imageRef string, imageID string) (v1alpha1.Regist
 	}
 	return registry, artifact, nil
 }
+
+func (p *plugin) parseOSRef(reports ty.Report) (v1alpha1.OS, error) {
+	os := v1alpha1.OS{}
+	if reports.Metadata.OS != nil {
+		os.Family = reports.Metadata.OS.Family
+		os.Name = reports.Metadata.OS.Name
+		eosl := reports.Metadata.OS.Eosl
+		if eosl {
```

Comment:
So Trivy doesn't always include an OS section in reports. There's three different responses I've seen:

OS section is returned and EOSL is true
```json
{
  "SchemaVersion": 2,
  "ArtifactName": "nginx:1.16-alpine",
  "ArtifactType": "container_image",
  "Metadata": {
    "OS": {
      "Family": "alpine",
      "Name": "3.10.4",
      "EOSL": true
    },...
 }
```
OS section is returned and no EOSL response
```json
{
  "SchemaVersion": 2,
  "ArtifactName": "redis:7.2.2-alpine3.18",
  "ArtifactType": "container_image",
  "Metadata": {
    "OS": {
      "Family": "alpine",
      "Name": "3.18.4"
    },...
 }
```

No OS section is returned
```json
{
  "SchemaVersion": 2,
  "ArtifactName": "rancher/coredns-coredns:1.8.3",
  "ArtifactType": "container_image",
  "Metadata": {
    "ImageID": "sha256:3885a5b7f138c3e8779a18f672fd55c3c7cf0166b67cf5143ab0cd137eb38418",
    "DiffIDs": [
      "sha256:225df95e717ceb672de0e45aa49f352eace21512240205972aca0fccc9612722",
      "sha256:85c53e1bd74e42c7decc4f7362eca51cd6cba15c227a75b3d90e8d01e8bb97e7"
    ],
    "RepoTags": [
      "rancher/coredns-coredns:1.8.3"
    ],...
 }
```
I originally had it where EOSL was true/false however once I discovered the responses with no OS section I decided it was probably more accurate to leave it as empty since I don't have the information. 

I'm open to changing it back to true/false if desired.